fast_com.py

```diff
@@ -1,222 +1,194 @@
-#!/usr/bin/env python
+#!/usr/bin/env python3
 
 '''
-Python CLI-tool (without need for a GUI) to measure Internet speed with fast.com
-
+Python CLI-tool to measure Internet speed with fast.com
 '''
 
 
```

Comment:
I suggest adding `from __future__ import print_function, division` in order to get consistent results across python 2 and 3.